test/Chainweb/Test/Pact/Checkpointer.hs

```diff
@@ -605,17 +605,17 @@ runExec :: CheckpointEnv -> PactDbEnv'-> Maybe Value -> Text -> IO EvalResult
 runExec cp (PactDbEnv' pactdbenv) eData eCode = do
     execMsg <- buildExecParsedCode Nothing {- use latest parser version -} eData eCode
     evalTransactionM cmdenv cmdst $
-      applyExec' defaultInterpreter execMsg [] h' permissiveNamespacePolicy
+      applyExec' 0 defaultInterpreter execMsg [] h' permissiveNamespacePolicy
   where
     h' = H.toUntypedHash (H.hash "" :: H.PactHash)
     cmdenv = TransactionEnv Transactional pactdbenv (_cpeLogger cp) def
-             noSPVSupport Nothing 0.0 (RequestKey h') 0 def
+             noSPVSupport Nothing 0.0 (RequestKey h') 0 (mkExecutionConfig [FlagDisablePact44])
```

Comment:
This is a bit weird. Shouldn't we be able to test the checkpointer with the newest Pact?